runtime/src/server/middleware/get_page_renderer.ts

```diff
@@ -0,0 +1,402 @@
+import { writable } from 'svelte/store';
```

Comment:
I think renaming this file as part of this PR makes the diff too hard to follow

Reply:
You're right! I moved `get_page_renderer` back into `get_page_handler` to make the diff more readable (which probably also reduces the risk of merge conflicts)